pkgs/applications/misc/tty-share/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+# Upstream has a `./vendor` directory with all deps which we rely upon.
+buildGoPackage rec {
+  pname = "tty-share";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "elisescu";
+    repo = "tty-share";
+    rev = "v${version}";
+    sha256 = "1d2vd3d1lb4n0jq4s0p5mii1vz4r3z36hykr5mnx53srsni1wsj5";
+  };
+
+  goPackagePath = "github.com/elisescu/tty-share";
+
+  meta = with stdenv.lib; {
+    homepage = "https://tty-share.com";
+    description = "Share terminal via browser for remote work or shared sessions";
+    platforms = platforms.linux;
+    license = licenses.mit;
+    maintainers = [ andys8 ];
```

Something is wrong with ofborg which detects these issues automatically:

```suggestion
    maintainers = with maintainers; [ andys8 ];
```

---

There are other packages that have it like this too, but I can change it.

---

> There are other packages that have it like this too

Impossible, it doesn't pass ofborg's evaluations, and it doesn't pass `nixpkgs-review`.